apps/test-suite/tests/SQLite.js

```diff
@@ -508,6 +508,19 @@ export function test(t) {
         });
       });
 
+      t.it('should load crsqlite extension correctly', async () => {
+        const db = SQLite.openDatabase('test.db');
+        await db.execAsync([{ sql: 'DROP TABLE IF EXISTS foo;', args: [] }], false);
```

Comment:
nit: technically you should

```
SELECT crsql_begin_alter('foo');
DROP TABLE foo;
SELECT crsql_commit_alter('foo');
```

when modifying crrs. The `select` to start modification is an eye-sore but something that I'll address in https://github.com/vlcn-io/cr-sqlite/issues/181